packages/store/ts/common.ts

```diff
@@ -14,6 +14,12 @@ export type TableRecord<TKeySchema extends KeySchema = KeySchema, TValueSchema e
   value: SchemaToPrimitives<TValueSchema>;
 };
 
+export type ConfigFieldTypeToSchemaAbiType<T extends FieldData<string>> = T extends SchemaAbiType
+  ? T
+  : T extends `${string}[${string}]`
+  ? "uint8[]"
+  : "uint";
```

Comment:
is this to turn enums into schema types? should line 21 be `uint8`?
```suggestion
export type ConfigFieldTypeToSchemaAbiType<T extends FieldData<string>> = T extends SchemaAbiType
  ? T
  : T extends `${string}[${string}]`
  ? "uint8[]"
  : "uint8";
```

Reply:
good catch!